src/main/java/mezz/jei/gui/overlay/IngredientListOverlay.java

```diff
@@ -304,10 +304,18 @@ public boolean onGlobalKeyPressed(int eventKey) {
 				Config.toggleCheatItemsEnabled();
 				return true;
 			}
+			if(KeyBindings.toggleEditMode.isActiveAndMatches(eventKey)) {
```

Comment:
Please keep the `if (` style with a space between `if` and `(`

Reply:
Fixed in [18243e31](https://github.com/mezz/JustEnoughItems/pull/1530/commits/18243e31cfb436d520ca9bceb2fe57c84aee58e7)